example_byteslice_littleendian_test.go

```diff
@@ -9,3 +9,10 @@ func ExampleRSet() {
 	fmt.Printf("%x\n", RSet(data, setData))
 	// Output: bada55
 }
+
+func ExampleRUnset() {
+	data := []byte{0xDA, 0x99, 0xBA}
+	unsetData := []byte{0xAD, 0x11, 0xAB}
```

Comment:
**unsetData** can be shorter or longer than **data** but not the same length.
If not, the usage is pretty similar of `Unset` function.